base/abstractarray.jl

```diff
@@ -1248,11 +1248,11 @@ function _setindex!(::IndexCartesian, A::AbstractArray{T,N}, v, I::Vararg{Int, N
 end
 function _setindex!(::IndexCartesian, A::AbstractArray, v, I::Vararg{Int,M}) where M
     @_inline_meta
```

Comment:
```suggestion
    @_propagate_inbounds_meta
```

Reply:
Yes. It's needed for bootstrapping reasons: `@propagate inbounds` needs functionality that isn't yet defined at this stage of building julia. Since macro expansion runs immediately, that would trigger a build error. `@_propagate_inbounds_meta` is much simpler and can be defined at an earlier stage.